architecture-evaluation/src/main/java/edu/kit/kastel/sdq/case4lang/refactorlizar/architecture_evaluation/graphs/CtTypeSystemGraphUtils.java

```diff
@@ -6,17 +6,17 @@
 import spoon.Launcher;
 import spoon.reflect.declaration.CtType;
 
-public class SystemGraphs {
+public class CtTypeSystemGraphUtils implements SystemGraphUtils<CtType<?>> {
 
-    private SystemGraphs() {}
+    public CtTypeSystemGraphUtils() {}
 
-    public static MutableGraph<Node> convertToSystemGraph(Graph<Node> graph) {
-        MutableGraph<Node> systemGraph = Graphs.copyOf(graph);
+    public MutableGraph<Node<CtType<?>>> convertToSystemGraph(Graph<Node<CtType<?>>> graph) {
+        MutableGraph<Node<CtType<?>>> systemGraph = Graphs.copyOf(graph);
         // add empty node as system node
         CtType<?> type = Launcher.parseClass("class ThIsIsRiskant {}");
         var method = new Launcher().createFactory().createMethod();
         type.addMethod(method);
-        systemGraph.addNode(new Node(method));
+        systemGraph.addNode(new SpoonNode(method));
         return systemGraph;
     }
```

Comment:
warning: [MissingOverride] convertToSystemGraph implements method in SystemGraphUtils; expected @Override
    public MutableGraph<Node<CtType<?>>> convertToSystemGraph(Graph<Node<CtType<?>>> graph) {

Reply:
It shows that I did these changes with an unconfigured Eclipse workspace (for Kieker we have instructions for that :-)). Added the @override.